tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -317,7 +317,11 @@ public:
       return std::make_unique<RMergeableFill<Hist_t>>(*fResultHist);
    }
 
-   std::string GetActionName() { return "FillWithUnknownAxes"; }
+   std::string GetActionName()
```

Comment:
`FillHelper` is not only responsible for the `Histo1D` action

Reply:
Thanks for noting, @vepadulano . I was not completely sure about this line.

However, I was confused by line 242: `const std::shared_ptr<Hist_t> fResultHist;` (this led me to the conclusion that I will be creating histograms only). And also I tried creating other `TH1` objects such as `Histo2D`, `Profile1D`, `Profile2D`, but they were all calling `GetActionName()` from line 508.

So my questions now are:
1. What would be the best name for this action? To remain `"FillWithUnknownAxes"` as before? (And below to have the name and the number of entries)
2. What is another object type that can enter this condition? (I didn't discover one manually)

Reply:
`FillHelper` is only for `Histo1D` and no binning specified. `FillParHelper` is used otherwise.

Reply:
Ah, I see! Thanks for the clarification

Reply:
No prob, these names are not great